src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1406,4 +1406,19 @@ This version used in: {2}</value>
   <data name="Convert_to_linq_call_form" xml:space="preserve">
     <value>Convert to LINQ (call form)</value>
   </data>
+  <data name="Remove_unused_member" xml:space="preserve">
+    <value>Remove unused member</value>
+  </data>
+  <data name="Remove_Unused_Private_Members_Message" xml:space="preserve">
+    <value>Type '{0}' has an unused private member '{1}' which can be removed.</value>
```

Comment:
message ID does not match message value.  It should be `Type_0_has_an_unused_private_member_1_which_can_be_removed`.  The benefit here is that it's immediately obvious what it means in code.  And it also is clear how many args it needs when constructed.

Reply:
This is also the IDE pattern.  So please do not deviate unnecessarily :)  Thanks!